test/system/users_test.rb

```diff
@@ -236,4 +236,18 @@ class UsersTest < ApplicationSystemTestCase
     visit current_path
     assert_link(href: 'https://discord.com/channels/715806612824260640/123456789000000007')
   end
+
+  test 'show daily report download button' do
+    kimura = users(:kimura)
+
+    visit_with_auth "/users/#{kimura.id}", 'komagata'
```

Comment:
```suggestion
    visit_with_auth "/users/#{users(:kimura).id}", 'komagata'
```

何度か使ってるわけではないのでこれでいいかもです〜